java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -56,7 +56,9 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.set(key2, value2, SetOptions.builder().returnOldValue(true).build());
         baseTransaction.strlen(key2);
-        baseTransaction.customCommand(new String[] {"MGET", key1, key2});
+        baseTransaction.append(key2, value2);
+        baseTransaction.mset(Map.of(key2, value2));
```

Comment:
Why you do `mset`? to restore the value?
You can set `set` too, but this is not a mistake.

Reply:
it's more similar to the Python tests like this.

Reply:
Would it not be better to just at the end of the test but before `flush`? That way we don't add to unnecessary checks and it also won't modify keys needed for other tests.